franka_msgs/srv/ResetJointStates.srv

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
FYI: for these simple services you are usually good with using [std_srvs/Trigger](http://docs.ros.org/en/api/std_srvs/html/srv/Trigger.html). No need to change, but rather go with a `franka_msgs/SetJointConfiguration` instead

Reply:
Ah, I overlooked that message. Thanks for letting me know.